src/components/SingleArticle.jsx

```diff
@@ -7,20 +7,22 @@ function SingleArticle() {
   let { articleId } = useParams();
 
   const [comments, setComments] = useState([]);
+  const [showComments, setShowComments] = useState(false);
```

Comment:
You will need to implement some kind if indication for the user that the article is being loaded. Preferably  one for the comments as well.